docs/docs/deploying-to-digitalocean-droplet.md

```diff
@@ -53,15 +53,15 @@ Follow these instructions for installs on an Ubuntu droplet.
    sudo n stable
    ```
 
-   `Note`: If you check the version now, you would see the older versions of node and npm from the cache.
-   You can either exit and restart your terminal or refresh the cache by following commands,
+   > **Note:** If you check the version now, you would see the older versions of Node.js and npm from the cache.
```

Comment:
We're not at all consistent in our treatment of notes like this but I do agree that the backticks should be removed. As long as they're consistent within a single page, I think it's fine!

Reply:
we should have a unified syntax for all notes and hints etc in a styleguide

for example in the tutorial it is also not equal over tutorial steps 1-8 :( (#22205)